lbrynet/core/Wallet.py

```diff
@@ -1321,6 +1321,23 @@ def get_address_balance(self, address, include_balance=False):
         else:
             return Decimal((float(c) + float(u) + float(x)) / COIN)
 
+    @defer.inlineCallbacks
+    def create_addresses_with_balance(self, num_addresses, amount, broadcast=True):
+        addresses = self.wallet.get_unused_addresses(account=None)
+        if len(addresses) > num_addresses:
+            addresses = addresses[:num_addresses]
+        elif len(addresses) < num_addresses:
+            for i in range(len(addresses), num_addresses):
+                address = self.wallet.create_new_address(account=None)
+                addresses.append(address)
+            yield self._save_wallet()
```

Comment:
Can you add a TODO to remove `_save_wallet` from lbrynet? (lbryum should take care of it)